variables.tf

```
@@ -36,6 +36,12 @@ variable "ca_certificate_secret" {
   type        = string
 }
 
+variable "labels" {
+  type        = map(string)
+  description = "Labels to apply to resources"
+  default     = { "label" : "label" }
```

I think that an empty default would be less surprising for users.

```suggestion
  default     = {}
```